charts/gardener/controlplane/charts/application/templates/apiservice-v1alpha1-security-gardener-cloud.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apiregistration.k8s.io/v1
 kind: APIService
 metadata:
-  name: v1alpha1.authentication.gardener.cloud
+  name: v1alpha1.security.gardener.cloud
```

Comment:
You could add a note somewhere in `pkg/apis/authentication/v1alpha1/doc.go` that then authentication API is not served by the gardener-apiserver and does not have a corresponding APIService. To prevent others to hit the same issue in the future.

Reply:
I added such note on few notable places with 03ed22b,  that should be visible when someone tries to register the API to be served by the server, hope that is good enough.